launcher/cloudbuild.yaml

```diff
@@ -1,8 +1,6 @@
 substitutions:
-  # using this base image for now, because there is an issue causing the newest COS dev
-  # image not booting with cs.
-  '_BASE_IMAGE': 'cos-dev-117-18374-0-0' # Use a stable COS version to avoid CS image build issues
-  '_BASE_IMAGE_FAMILY': 'cos-dev' # base image family
+  '_BASE_IMAGE': '' # empty meansing using latest image in the below image family
```

Comment:
An empty base image means the build will use the latest image in '_BASE_IMAGE_FAMILY'